neurons/nodes/bitcoin/node.py

```diff
@@ -151,6 +151,15 @@ def create_challenge(self, start_block_height, last_block_height):
         challenge = Challenge(in_total_amount=in_total_amount, out_total_amount=out_total_amount, tx_id_last_4_chars=txn_id[-4:])
         return challenge, txn_id
 
+    def validate_challenge_response_output(self, challenge: Challenge, response_output):
+        if response_output[-4:] != challenge.tx_id_last_4_chars:
```

Comment:
with 4 characters its likely that we will get multimple results and getting first one from many will lead to same errors

how baout 5 or 6 chars instead?

Reply:
even if we change it to 5 or 6 characters, we won't completely eliminate the possibility of getting multiple results for a given challenge.
this PR already contains the validator logic that validates miners' responses even if it's different than the expected response. so staying at 4 won't cause any more errors.
and changing to 5 or 6 characters require updates from both miners and validators and it might cause fast updating miners to get failed for challenge.